src/main/kotlin/no/nav/syfo/personstatus/db/getFromPersonOversiktStatus.kt

```diff
@@ -54,6 +54,7 @@ const val queryHentUbehandledePersonerTilknyttetEnhet = """
                             OR trenger_oppfolging = 't'
                             OR behandler_bistand_ubehandlet = 't'
                             OR arbeidsuforhet_vurder_avslag_ubehandlet = 't'
+                            OR (friskmelding_til_arbeidsformidling_fom IS NOT NULL AND friskmelding_til_arbeidsformidling_fom >= CURRENT_DATE)
```

Comment:
Må kanskje legge til denne betingelsen andre steder vi henter personer med som skal vises i oversikten også? Cronjob for å oppdatere enhet og virksomhetsnavn er det jeg kommer på i farten.

Reply:
Ja, det er sant - glemte det.